features/lorawan/LoRaWANStack.h

```diff
@@ -70,7 +70,8 @@ class LoRaWANStack: private mbed::NonCopyable<LoRaWANStack> {
     } device_states_t;
 
 public:
-    static LoRaWANStack& get_lorawan_stack();
+    LoRaWANStack();
+    ~LoRaWANStack();
```

Comment:
I can't see any reason to be defining an empty destructor here. Yes, it's an existing pattern, but tidy up?

Reply:
Ok, I'll remove the empty destructor as this class is not supposed to be herited.